### DISCUSSION TITLE
When using a custom action with the Attach action, `Attach & attach another`button does not work.

### DISCUSSION THREAD

**Discussion by bzy107** [filamentphp/filament#9625] [Help]
In the scenario where processing is done with a custom form and custom action using the AttachAction as shown below, even when attaching using `Attach & attach another`, the modal closes. 

Is there a solution?
```php
            ->headerActions([
                Tables\Actions\AttachAction::make()
                    ->form()
                    ->action()
```
![スクリーンショット 2023-11-14 3 37 46](https://github.com/filamentphp/filament/assets/28666304/720dd79e-0dd8-4df7-852e-22bbca5b2b56)

**Comment by sinnbeck** [comment]
Take a look at the demo. It has this feature as well
https://demo.filamentphp.com/blog/authors

Click "New author"

The complete source is available here
https://github.com/filamentphp/demo

**Reply by bzy107** [reply]
What I am seeking is a resolution to the issue where consecutive attaches are not possible when customizing the attachAction behavior, not the createAction. Thank you
